victims/models.py

```diff
@@ -48,3 +61,15 @@ def tags_and_colors(self):
 
     def is_valid(self):
         return bool(self.when)
+
+    def to_json(self):
+        items = {
+            "id": self.id,
+            "aggressor_side": self.aggressor_side,
+            "when": str(self.when),
+            "state": self.state,
+            "city": self.city,
+            "tags": self.tags,
+            "stories": [story.to_JSON() for story in self.stories],
```

Comment:
This line should not encode `story` as JSON since the whole `items` is encoded in the end.

Reply:
good point, thanks

Reply:
@baltazarix do you mind adding a simple test for this endpoint? [It's pretty simple](https://sanic.readthedocs.io/en/latest/sanic/testing.html) and can avoid assure we're serializing data correctly.

Reply:
@turicas, sure! I was thinking about it but I've had no idea how test scenario should look like. Only ideas I have are:
- checking for status code
- create some test data (e.g. 1 `Case` instance, 2 `Story` instances), then compare response content.

Feel free to tell me if I'm missing something :)

Reply:
> Take a look at existing `test_home_contents`: we test some contents that we expect to see in the response. Name of the stories, maybe dates formatted as str (as they are the bottleneck of this serialization) etc…

Yep, I wanted to make something similar, sorry if I wasn't clear enough.